src/accelerate/utils/imports.py

```diff
@@ -125,3 +128,26 @@ def is_sagemaker_available():
 
 def is_tqdm_available():
     return importlib.util.find_spec("tqdm") is not None
+
+
+def is_ipex_available():
+    def get_major_and_minor_from_version(full_version):
+        return str(version.parse(full_version).major) + "." + str(version.parse(full_version).minor)
+
+    _torch_version = importlib_metadata.version("torch")
+    if importlib.util.find_spec("intel_extension_for_pytorch") is None:
+        return False
+    _ipex_version = "N/A"
+    try:
+        _ipex_version = importlib_metadata.version("intel_extension_for_pytorch")
+    except importlib_metadata.PackageNotFoundError:
+        return False
+    torch_major_and_minor = get_major_and_minor_from_version()
```

Comment:
Should '_torch_version' be passed to the inner function get_major_and_minor_from_version()?